Sources/FoundationInternationalization/Predicate/LocalizedString.swift

```diff
@@ -12,7 +12,7 @@
 
 #if FOUNDATION_FRAMEWORK
 
-@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *)
+@available(Future, *)
```

Comment:
This will need to be 5.9 runtime dependent (along with the rest of the declarations in this file)

Reply:
Ahh my bad.... thanks for catching this!